tests/acceptance/cli_test.py

```diff
@@ -984,6 +984,37 @@ def test_run_without_command(self):
             [u'/bin/true'],
         )
 
+    def test_run_rm(self):
+        volumes = self.client.volumes()['Volumes']
+        if volumes is None:
+            volumesBefore = 0
+        else:
+            volumesBefore = len(volumes)
```

Comment:
I don't think this is the right way to test this. It's too prone to failure. It assumes the engine being used is not in use by anything else, which is not a good assumption. The test could also pass if no volume was created, even though the remove didn't work.

A better way to test this would be to inspect the container that is created, get the volume id, and assert that the volume doesn't exist when the container exits. It's a little more involved to write, but you can do it with `start_process()` and `wait_on_process()`.

You'll need to run a non-terminating command instead of `/bin/true`.

```python

start_process(...)
wait_on_condition(ContainerCountCondition(self.project, 1))
container, = self.project.get_service('test').containers(...)
self.dispatch(["stop"])

# inspect volume by id and assert a not found error
```